src/Context/WebApiContext.php

```diff
@@ -59,6 +59,14 @@ public function setClient(Client $client)
     }
 
     /**
+     * @return \GuzzleHttp\Message\ResponseInterface
+     */
+    public function getResponse()
```

Comment:
As far as I know it is not required outside of the scope of this class, in which case it is always better to encapsulate functionality.